shared/app.js

```diff
@@ -19,6 +19,7 @@ module.exports = Backbone.Model.extend({
 
   defaults: {
     loading: false,
+    templateEngine: 'handlebars',
     templateAdapter: 'rendr-handlebars'
```

Comment:
Makes it more verbose and complete solution. Allowing to share templating engine with non-Rendr parts of the application.